src/pds/aossrequestsigner/errors.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+
+class Non200HttpStatusError(RuntimeError):
+    def __init__(self, status_code: int, description: Optional[str] = None):
+        if status_code == 200:
```

Comment:
There's `http.HTTPStatus.OK` that can be used in place of `200` … but given that `200` appears in the name of the class—and "200" is pretty universally understood these days—maybe an explicit `200` is fine

Reply:
Better is betterrer - will fix.

Reply:
96b2934c